traitsui/qt4/ui_panel.py

```diff
@@ -1098,9 +1100,9 @@ def _create_label(self, item, ui, desc, suffix=':'):
         we append a suffix (by default a colon ':') at the end of the
         label text.
 
-        We also set the help on the QLabel control (from item.help) and
-        the tooltip (it item.desc exists; we add "Specifies " at the start
-        of the item.desc string).
+        We also set the help on the QLabel control (from item.help) and the
+        tooltip (it item.desc exists; we add "Specifies " at the start of the
+        item.desc string, if item.tooltip exists we just use it as is).
```

Comment:
I would talk about `tooltip` first rather than second, in order to mirror the code's logic. There are also a couple of errors in the original text that should be fixed. This is how I would write the parenthetical:

```
(if the ``tooltip`` metadata on the edited trait exists, then it will be used as-is;
otherwise, if the ``desc`` metadata exists, the string "Specifies " will be
prepended to the start of ``desc``)
```

Reply:
Thanks! Done.